<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatePrizeForm</class>
 <widget class="QWidget" name="CreatePrizeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Prize</string>
  </property>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>28</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Create Prize</string>
   </property>
   <property name="buddy">
    <cstring>PlaceNumberlineEdit</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Crate Prize</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>90</y>
     <width>233</width>
     <height>138</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Place Number</string>
      </property>
      <property name="buddy">
       <cstring>PlaceNumberlineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="PlaceNumberlineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Place Name</string>
      </property>
      <property name="buddy">
       <cstring>PrizePercentagelineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="PlaceNamelineEdit"/>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Prize Amount</string>
      </property>
      <property name="buddy">
       <cstring>PlaceNamelineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLineEdit" name="PrizeAmountlineEdit"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>- or -</string>
      </property>
      <property name="buddy">
       <cstring>PrizeAmountlineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Prize Percentage</string>
      </property>
      <property name="buddy">
       <cstring>PrizeAmountlineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="PrizePercentagelineEdit"/>
    </item>
    <item row="4" column="3">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>%</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
